src/t_zset.c

```diff
@@ -3963,12 +3962,14 @@ void blockingGenericZpopCommand(client *c, robj **keys, int numkeys, int where,
 
 // BZPOPMIN key [key ...] timeout
 void bzpopminCommand(client *c) {
-    blockingGenericZpopCommand(c,c->argv+1,c->argc-2,ZSET_MIN,c->argc-1,0,0);
+    blockingGenericZpopCommand(c, c->argv+1, c->argc-2, ZSET_MIN,
+                               c->argc-1, 0, 0);
```

Comment:
i see you added some line breaks, which i personally think are only necessary in extremely long lines, or complex ones with some comments.